Extension/ui/settings.html

```diff
@@ -455,7 +455,7 @@
             <tr>
             <!-- select configuration name -->
             <td>
-                <div class="select-editable" style="width: 300px; margin-right: 10px">
+                <div class="select-editable" style="width: 300px; margin-left: -3px; margin-right: 7px">

```

Comment:
`margin-left: -3px; margin-right: 7px` is this still aligned with the other selection boxes on the UI page?

Reply:
yes, there is a screenshot. Let me know